integreat_cms/cms/utils/pdf_utils.py

```diff
@@ -83,7 +83,11 @@ def generate_pdf(region, language_slug, pages):
             # In any other case, take the region name
             title = region.name
     language = Language.objects.get(slug=language_slug)
-    filename = f"{pdf_hash}/{capfirst(settings.BRANDING)} - {language.translated_name} - {title}.pdf"
+    # Make sure, that the length of the filename is valid
+    truncated_filename = Truncator(
+        f"{pdf_hash}/{capfirst(settings.BRANDING)} - {language.translated_name} - {title}"
+    ).chars(250, truncate="")
+    filename = f"{truncated_filename}.pdf"
```

Comment:
```suggestion
    ext = ".pdf"
    max_len = os.statvfs(settings.PDF_ROOT).f_namemax - len(ext)
    name = f"{capfirst(settings.BRANDING)} - {language.translated_name} - {title}"
    filename = f"{pdf_hash}/{name[:max_len]}{ext}"
```
Since the maximum filename length depends on the file system used by the OS, I guess it's better to ask the system instead of hardcoding the value (although 255 chars seems to be pretty standard...)